index.js

```diff
@@ -95,28 +96,25 @@ export const provideLinter = () => {
         options.syntax = 'scss';
       }
 
-      return new Promise((resolve) => {
-        cosmiconfig('stylelint', {
-          cwd: path.dirname(filePath)
-        }).then(result => {
-          if (result) {
-            options.config = assign(rules, result.config);
-            options.configBasedir = path.dirname(result.filepath);
-          }
-
-          resolve(runStylelint(options, filePath));
-        }).catch(error => {
-          atom.notifications.addWarning(`Invalid config file`, {
-            detail: error.message || `Failed to parse config file`,
-            dismissable: true
-          });
-
-          if (usePreset()) {
```

Comment:
It's not a good idea to re-lint with the preset if the config had errors. We would be linting the file without the complete set of rules the user wants.